content/spin/running-apps.md

```diff
@@ -104,6 +104,8 @@ command = "cargo build --target wasm32-wasi --release"
 watch = ["src/**/*.rs", "Cargo.toml"]
 ```
 
+Note that references to files in the `component.build.watch` section use the optional `component.build.workdir` section as the base directory for references. If not specified, it uses the directory of the `spin.toml` file. 
```

Comment:
I got a bit lost in the references to references - I reckon we could simplify the wording e.g.

```suggestion
If the `build` section specifies a `workdir`, then `watch` patterns are relative to that directory. Otherwise, `watch` patterns are relative to the directory containing the `spin.toml` file.
```